easybuild/easyconfigs/d/dask/dask-2023.7.1-foss-2022b.eb

```diff
@@ -0,0 +1,69 @@
+easyblock = 'PythonBundle'
+
+name = 'dask'
+version = '2023.7.1'
+
+homepage = 'https://dask.org/'
+description = """Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance
+ at scale for the tools you love."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('PyYAML', '6.0'),
+    ('SciPy-bundle', '2023.02'),
+    ('bokeh', '3.2.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('toolz', '0.12.0', {
+        'checksums': ['88c570861c440ee3f2f6037c4654613228ff40c93a6c25e0eba70d17282c6194'],
+    }),
+    ('locket', '1.0.0', {
+        'checksums': ['5c0d4c052a8bbbf750e056a8e65ccd309086f4f0f18a2eac306a8dfa4112a632'],
+    }),
+    ('partd', '1.4.0', {
+        'checksums': ['aa0ff35dbbcc807ae374db56332f4c1b39b46f67bf2975f5151e0b4186aed0d5'],
+    }),
+    ('HeapDict', '1.0.1', {
+        'checksums': ['8495f57b3e03d8e46d5f1b2cc62ca881aca392fd5cc048dc0aa2e1a6d23ecdb6'],
+    }),
+    ('zict', '3.0.0', {
+        'checksums': ['e321e263b6a97aafc0790c3cfb3c04656b7066e6738c37fffcca95d803c9fba5'],
+    }),
+    ('tblib', '2.0.0', {
+        'checksums': ['a6df30f272c08bf8be66e0775fad862005d950a6b8449b94f7c788731d70ecd7'],
+    }),
+    ('versioneer', '0.29', {
+        'checksums': ['5ab283b9857211d61b53318b7c792cf68e798e765ee17c27ade9f6c924235731'],
+    }),
+    (name, version, {
+        'checksums': ['bec23e3bfed427d86035f3ecfc1cd1ee6ef375ba40bfb1f8a5f03ad3f2fdbbba'],
+    }),
+    ('distributed', version, {
+        'checksums': ['66442f6af32e03815de6d3a5251f8ad9bbd63bb018c48aa361efca5355e46749'],
+    }),
+    ('dask-mpi', '2022.4.0', {
+        'checksums': ['0a04f1d7d35a06cdff506593330d4414ea242c9172498ce191f5742eac499e17'],
+    }),
+    ('docrep', '0.3.2', {
+        'checksums': ['ed8a17e201abd829ef8da78a0b6f4d51fb99a4cbd0554adbed3309297f964314'],
+    }),
+    ('dask-jobqueue', '0.8.2', {
+        'checksums': ['d35407a05a0534347c5d958ae749b9f8535bec529857d013b6e5649cde43914a'],
+    }),
+]
```

Comment:
I see that in `dask-2022.10.0-foss-2022a.eb` `fsspec` and `cloudpickle` were also in the ext list. Just checking: I guess you removed those on purpose, and they are not needed, correct?